### ISSUE TITLE
Unable to use BCM 7346 tuner on ZGEMMA H2S

### ISSUE THREAD

**Issue by mark-royds** [catalinii/minisatip#900]
I have a ZGEMMA H2S box that has two DVB-S2 tuners.  
The tuners are
- frontend0 BCM 7346 internal
- frontend1 Avla 6211 

The box is running a recent openatv image (I've also tried other images but the symptoms are the same).
Enigma2 is able to successfully tune on both tuners.

After killing all enigma2 processes, I launch minisatip as follows:
minisatip --logfile /home/root/minisatip.log -v adapter -R /usr/share/minisatip/html

I'm using VLC on another machine on the LAN to access a FTA signal.
I can successfully tune only if I include "&fe=2" in the rtsp URI to force minisatip to use the Avla tuner.  If I don't specify "fe" or use "&fe=1" so that BCM 7346 is used, tuning fails.

In the minisatip web interface when using fe=1, I can see that the tuner is attempting to tune, but no signal is ever shown.

I've poked around a bit... lsdvb (without e2 or minisatip running) shows a strange result:
```
root@zgemmah2s:~# lsdvb

                lsdvb: Simple utility to list PCI/PCIe DVB devices
                Version: 0.0.4
                Copyright (C) Manu Abraham

 (0:1997731104 1996464824:1997781984) on PCI Domain:4198804 Bus:1 Device:0 Function:4298352
        DEVICE:1 ADAPTER:0 FRONTEND:0 (DVB-S2)
                 FE_QPSK Fmin=950MHz Fmax=2150MHz

 (0:1997731104 1996464824:1997781984) on PCI Domain:4198804 Bus:1 Device:0 Function:4298352
        DEVICE:0 ADAPTER:0 ERROR: Open frontend0 failed
```

I can't see anything holding that tuner:
```
root@zgemmah2s:~# lsof | grep -i dvb
root@zgemmah2s:~# 
```

It's a mystery to me

**Comment by mark-royds** [comment]
[minisatip.log](https://github.com/catalinii/minisatip/files/7357379/minisatip.log)
Line 123 - Start tune on FE2
Line 729 - Start tune on FE1

**Comment by mark-royds** [comment]
Spent a few hours learning the subtleties of mips32 vs mips32r2 cross-compilation... managed to get a working build with DVBAPI3.  Unfortunately, same issue.  FE1 cannot tune.

Logfile from the DVBAPI3 build attached
[minisatip-dvbapi3.log](https://github.com/catalinii/minisatip/files/7358529/minisatip-dvbapi3.log)

**Comment by catalinii** [comment]
I use https://hub.docker.com/r/catalinii/minisatip-build-image for cross compiling the binary using GH Actions

**Comment by lars18th** [comment]
Hi @mark-royds ,

Until you find a solution, you can continue using minisatip with only one tuner. Start it with `-e 1`, then only the tuner 1 (0 is the BCM one) will be in use, and you can do regular requests without adding the `&fe=2` parameter to use the Avla one.

For sure this is not a permanent solution, but it's a workaround to use what really works.

In the other hand, perhaps the solution is inside the source code of the E2. Check if other tools like mumudvb, vlc, etc. can tune or not when running them in your box.

I hope this helps.
Regards.

**Comment by mark-royds** [comment]
Hi @lars18th 

You nailed it.

Taking your advice I started stripping out tuning parameters from the request URI and was able to get the BCM tuner to lock if I removed the FEC parameter.  The minimal URL that worked was:
`rtsp://10.150.213.124/?fe=fe=1&src=1&freq=10847&pol=v&msys=dvbs2&pids=0,17,18,260,5400,5402&mtype=8psk&sr=23000`

Doing a bit of digging (KingOfSat) I discovered that the correct FEC for that transponder is 3/4 and not 2/3.  I'd taken the tuning parameters from the satip.info M3U channel lists and foolishly assumed it to be accurate.

The Avla tuner is clearly a lot more forgiving about incorrect parameters than the BCM.
